src/python/grapl-common/grapl_common/env_helpers.py

```diff
@@ -55,7 +55,9 @@ def _client_get(client_create_fn: Callable[..., Any], params: ClientGetParams) -
 
     # Unlike Rust FromEnv, we rely on boto3's built in region handling.
 
-    if all((endpoint_url, access_key_id, access_key_secret)):
+    if _running_in_localstack():
+        return _localstack_client(client_create_fn, params)
+    elif all((endpoint_url, access_key_id, access_key_secret)):
```

Comment:
Is this necessary? It looks like some of this is already covered by: https://github.com/grapl-security/grapl/blob/main/local-grapl.env#L88. This adding back some "IS_LOCAL"-like code, and it seems like we don't need to, but maybe I'm missing something.

Reply:
It's adding "is localstack" code, but leaving everything else as it was. Since this is shared code, we don't want to break the other things that are using the `IS_LOCAL` flag. Once everything has been migrated over, however, we can come back and yank a bunch of this code (though the localstack-stuff would need to stay for now, absent adding some custom DNS and proxy magic to our docker setup.